api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -1050,12 +1055,18 @@ definitions:
     properties:
       name:
         type: string
+      author:
+        type: string
       description:
         type: string
       version:
         type: string
       url:
         type: string
+      logo:
+        type: string
+      massastationVersion:
```

Comment:
its weird to find massaStationVersion here.
it is not a property of a plugin.
Or maybe its the massaStation minimum version required ? in that case its poorly named

Reply:
it is part of th manifest https://github.com/massalabs/thyra-plugin-store/blob/04a13e300f78a704e23308413ed8a6315ce68e4e/plugins.json#L5
but not really used in the front end, I can remove it

Reply:
ok! so its the minimum required version. can you just rename this ?
by instance `massastationMinVersion`